repos/spack_repo/builtin/packages/minimap2/package.py

```diff
@@ -47,4 +56,6 @@ def install_minimap2(self):
         make(*make_arg)
         mkdirp(prefix.bin)
         install("minimap2", prefix.bin)
+        if self.spec.satisfies("js_engine=node"):
+            filter_file(r"k8", "node", "./misc/*.js")
```

Comment:
Adam is correct. You cannot use this syntax for the files. Instead you can import `glob` to create a list of matching files , e.g. in variable `files` and pass the list as `*files` in the third argument.